pkg/report/bsd.go

```diff
@@ -50,12 +50,7 @@ func (ctx *bsd) ContainsCrash(output []byte) bool {
 }
 
 func (ctx *bsd) Parse(output []byte) *Report {
-	stripped := bytes.Replace(output, []byte{'\r', '\n'}, []byte{'\n'}, -1)
-	stripped = bytes.Replace(stripped, []byte{'\n', '\r'}, []byte{'\n'}, -1)
-	for len(stripped) != 0 && stripped[0] == '\r' {
-		stripped = stripped[1:]
-	}
-	rep := simpleLineParser(stripped, ctx.oopses, nil, ctx.ignores)
+	rep := simpleLineParser(output, ctx.oopses, nil, ctx.ignores)
```

Comment:
I think this can now be replaced with just `return simpleLineParser`.
We messed with rep.Output because what we passed to simpleLineParser wasn't real output.

Reply:
fixed.